frontend/app/routes/protected/person-case/parent-details.tsx

```diff
@@ -277,25 +287,28 @@ function ParentInformation({ defaultFormValues, errors }: ParentInformationProps
 interface ParentFormProps {
   index: number;
   defaultValues?: FormData;
+  loaderData: Route.ComponentProps['loaderData'];
   errors?: Record<string, [string, ...string[]] | undefined>;
   onRemove?: (index: number) => void;
 }
 
-function ParentForm({ index, defaultValues, errors, onRemove }: ParentFormProps) {
+function ParentForm({ index, defaultValues, loaderData, errors, onRemove }: ParentFormProps) {
```

Comment:
Since `defaultFormValues` is included in `loaderData`, does it make sense to just use the latter here?